lib/src/enum/user_property.dart

```diff
@@ -11,5 +11,5 @@ enum UserProperty {
   aiVerbosity,
   googleEmail,
   appleEmail,
-  ;
+  hasMetaGlasses;
```

Comment:
Nit: should have kept the `;` as the last line, and insert `hasMetaGlasses,` above that. This way PRs look cleaner with a single line change. And next time, you wouldn't have to change `hasMetaGlasses;` into `hasMetaGlasses,` before adding your new property.